packages/theme/src/notify/responsive.less

```
@@ -0,0 +1,35 @@
+@import '../custom.less';
+@import './vars.less';
+
+@notify-prefix-cls: ~'@{css-prefix}notify';
+
+@media screen and (max-width: 1280px) {
+  .@{notify-prefix-cls} {
```

Stylelint (17.4.0)

[error] Unknown word notify-prefix-cls (CssSyntaxError)

(CssSyntaxError)